### ISSUE TITLE
Assertion failing on boot

### ISSUE THREAD

**Issue by Ecco** [numworks/epsilon#1172]
#### Describe the bug
When launching the Emscripten simulator, an assertion fails.

```
12.0.0 4b9e42d
epsilon.js:8 Assertion failed: SharedStaticPool == nullptr, at: poincare/include/poincare/tree_pool.h,24,RegisterPool
```

#### Note

Our CI should have caught this. Investigate why it's not been the case.

**Comment by Ecco** [comment]
Here's a full log of master on a DEBUG build:

```
epsilon.js:8 emscripten_set_main_loop_timing: Cannot set timing mode for main loop since a main loop does not exist! Call emscripten_set_main_loop first to set one up.
_emscripten_set_main_loop_timing @ epsilon.js:8
epsilon.js:37 -12
epsilon.js:37 -12
abort @ epsilon.js:37
epsilon.js:8 warning: build with  -s DEMANGLE_SUPPORT=1  to link in libcxxabi demangling
warnOnce @ epsilon.js:8
epsilon.js:37 exception thrown: abort(-12) at Error
    at jsStackTrace (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:684446)
    at stackTrace (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:684617)
    at abort (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:37:46519)
    at _IonSimulatorCallbackDidScanKeyboard (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:30:127259)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:44592)
    at __ZN3Ion8Keyboard4scanEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:21:243413)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:19005)
    at __ZN3Ion6Events8getEventEPi (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:17:48228)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:30572)
    at __ZN7RunLoop4stepEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:19:142856)
This error happened during an emterpreter-async operation. Was there non-emterpreted code on the stack during save (which is unallowed)? If so, you may want to adjust EMTERPRETIFY_BLACKLIST, EMTERPRETIFY_WHITELIST. For reference, this is what the stack looked like when we tried to save it: 1,Error
    at Object.handle (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:887163)
    at _emscripten_sleep (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:887567)
    at _IonSimulatorCallbackDidScanKeyboard (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:30:127243)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:44592)
    at __ZN3Ion8Keyboard4scanEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:21:243413)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:19005)
    at __ZN3Ion6Events8getEventEPi (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:17:48228)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:30572)
    at __ZN7RunLoop4stepEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:19:142856)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:42293)
callMain @ epsilon.js:37
epsilon.js:8 Uncaught abort(-12) at Error
    at jsStackTrace (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:684446)
    at stackTrace (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:684617)
    at abort (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:37:46519)
    at _IonSimulatorCallbackDidScanKeyboard (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:30:127259)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:44592)
    at __ZN3Ion8Keyboard4scanEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:21:243413)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:19005)
    at __ZN3Ion6Events8getEventEPi (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:17:48228)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:30572)
    at __ZN7RunLoop4stepEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:19:142856)
This error happened during an emterpreter-async operation. Was there non-emterpreted code on the stack during save (which is unallowed)? If so, you may want to adjust EMTERPRETIFY_BLACKLIST, EMTERPRETIFY_WHITELIST. For reference, this is what the stack looked like when we tried to save it: 1,Error
    at Object.handle (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:887163)
    at _emscripten_sleep (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:887567)
    at _IonSimulatorCallbackDidScanKeyboard (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:30:127243)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:44592)
    at __ZN3Ion8Keyboard4scanEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:21:243413)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:19005)
    at __ZN3Ion6Events8getEventEPi (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:17:48228)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:30572)
    at __ZN7RunLoop4stepEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:19:142856)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:42293)
epsilon.js:37 'Pointer_stringify' was not exported. add it to EXTRA_EXPORTED_RUNTIME_METHODS (see the FAQ)
epsilon.js:37 'Pointer_stringify' was not exported. add it to EXTRA_EXPORTED_RUNTIME_METHODS (see the FAQ)
abort @ epsilon.js:37
epsilon.js:37 Uncaught abort('Pointer_stringify' was not exported. add it to EXTRA_EXPORTED_RUNTIME_METHODS (see the FAQ)) at Error
    at jsStackTrace (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:684446)
    at stackTrace (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:684617)
    at abort (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:37:46519)
    at Module.Pointer_stringify (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:37:39213)
    at https://www.numworks.com/simulator/update.js:4:23
This error happened during an emterpreter-async operation. Was there non-emterpreted code on the stack during save (which is unallowed)? If so, you may want to adjust EMTERPRETIFY_BLACKLIST, EMTERPRETIFY_WHITELIST. For reference, this is what the stack looked like when we tried to save it: 1,Error
    at Object.handle (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:887163)
    at _emscripten_sleep (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:8:887567)
    at _IonSimulatorCallbackDidScanKeyboard (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:30:127243)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:44592)
    at __ZN3Ion8Keyboard4scanEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:21:243413)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:19005)
    at __ZN3Ion6Events8getEventEPi (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:17:48228)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:30572)
    at __ZN7RunLoop4stepEv (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:19:142856)
    at emterpret (file:///Users/romain/Sources/epsilon/output/debug/simulator/web/simulator/epsilon.js:29:42293)
```

**Comment by EmilieNumworks** [comment]
I think this is not caught by the CI because the CI builds with DEBUG=0.

**Comment by EmilieNumworks** [comment]
I believe it was fixed by #1193. However, I could not reproduce the assertion failing.
